apps/desktop/src/lib/modes/service.ts

```diff
@@ -29,6 +40,20 @@ export class ModeService {
 
 		this.headAndMode.set({ head, operatingMode });
 	}
+
+	async enterEditMode(editeeCommitId: string, editeeBranchRef: string) {
+		await invoke('enter_edit_mode', {
+			projectId: this.projectId,
+			editee: editeeCommitId,
+			editeeBranch: editeeBranchRef
```

Comment:
At first glance I react to this variable name, I don't think it's a real word. What exactly does _editee_ refer to?

Reply:
<img width="746" alt="Screenshot 2024-08-19 at 12 01 54" src="https://github.com/user-attachments/assets/61424d6e-2a37-4c3b-af62-3a6d019e7e96">


I was calling the commit that was being edited the editee. I guess
```rs
struct EditMode {
  subject_commit: Oid,
  subject_branch: String
}
```
works. WDYT?

Reply:
Yeah I think that's better, but let me still ask, is a prefix necessary?
```rs
struct EditMode {
  commitId: Oid,
  branchId: String
}
```